test/suite_implementation.js

```diff
@@ -12,6 +17,7 @@ module.exports = function(style, options, _callback) {
         }
     }
 
+    window.XMLHttpRequest = XMLHttpRequest;
```

Comment:
Oh, I see, it's overwriting the sinon fake XHR. Can we use jsdom XHR by default and only substitute sinon for the unit tests?